src/UI/worldScreen/TileSidebar.ts

```diff
@@ -88,10 +88,16 @@ export default class TileSidebar implements Page {
         if (project.costs.length == 0) {
             projectHTML.appendChild(UI.makePara("Cost: Free"));
         } else {
-            const cssClass = this.run.inventory.canAfford(project.costs) ? "project-requirement-met" : "project-requirement-unmet";
-            const costDescriptions = project.costs.map((cost: Cost) => `${cost.toString()}`);
-            const costsString = "Cost: " + costDescriptions.join(", ");
-            projectHTML.appendChild(UI.makePara(costsString, [cssClass]));
+            const costParas = project.costs.map((cost: Cost) => UI.makePara(cost.toString(),
+                [
+                    this.run.inventory.canAfford([cost])
+                    ? "project-requirement-met"
+                    : "project-requirement-unmet"
+                ])
```

Comment:
the indentation here is not super readable, and the linter doesn't like the indentation. You can just collapse these 5 lines into 1 line, though, and it'll still fit in under 120 characters.